src/tasks/minions/minigames/nightmareZoneActivity.ts

```diff
@@ -0,0 +1,58 @@
+import { Monsters } from 'oldschooljs';
+
+import { resolveAttackStyles } from '../../../lib/minions/functions';
+import { incrementMinigameScore } from '../../../lib/settings/settings';
+import { SkillsEnum } from '../../../lib/skilling/types';
+import { handleTripFinish } from '../../../lib/util/handleTripFinish';
+import { NightmareZoneActivityTaskOptions } from './../../../lib/types/minions';
+
+export const nightmareZoneTask: MinionTask = {
+	type: 'NightmareZone',
+	async run(data: NightmareZoneActivityTaskOptions) {
+		const { quantity, userID, channelID, duration, strategy } = data;
+		const user = await mUserFetch(userID);
+
+		const [, , attackStyles] = resolveAttackStyles(user, { monsterID: Monsters.Cow.id });
```

Comment:
again, this makes no sense. its not even an easter egg. unless im missing something.